src/main/java/BasketInformations.java

```diff
@@ -0,0 +1,98 @@
+package com.company;
+
+import java.util.HashMap;
+
+/**
+ * This class is a basket
+ */
+public class BasketInformations {
+
+	// The product of the basket
+	static HashMap<String, Integer> map = new HashMap<String, Integer>();
```

Comment:
```suggestion
	private final Map<String, Double> productsMap = new HashMap();
```
- variable name: Since this field is supposed to store the products map belonging to the basket, I would like to suggest we change it to something easier to understand at a first look, something like `productsMap` .
- field modifier: the `map` field, since it has the default access modifier and also `static` means that we will have only one state per class and not one state per BasketInformations instances, also it can be reassigned by other classes from the same package, it may be set to `null` for example, to avoid reassigning we can use the `final` modifier.
- type: Since we are not using any feature specific from the HashMap implementation of Map, we could just declare it as `Map<X,Y>`, this way we are free to use any implementation, like a thread-safe one, and follow some conventions, nothing else.